Dockerfile

```diff
@@ -43,6 +47,13 @@ RUN \
       valgrind \
       wget \
       zlib1g-dev \
+  # Setup Node.js package repository
```

Comment:
Can we not use the one in the main repo?

Reply:
I can update the comment, the one in the main repo is old and doesn't install `npm`, and this is the officially recommended way to install node/npm.

I could do `nvm` instead probably ?

Reply:
ah today i learned. Been spoiled by arch i suppose